buttons.css

```
@@ -0,0 +1,26 @@
+.header__navigation__button {
+  background-color: rgb(134, 134, 134);
+  box-shadow: 0px 5px 20px rgb(74, 74, 74);
+  color: white;
+  padding: 15px;
+  margin: 5px;
+  border-radius: 15px;
+}
+
+.address__button {
+  background-color: rgb(86, 107, 88);
+  border-radius: 10px;
+  margin: 0px;
+  padding: 15px;
+  color: white;
+  box-shadow: 0px 5px 10px rgb(74, 74, 74);
+}
+
+.footer__navigation__button {
+  background-color: rgb(134, 134, 134);
+  box-shadow: 0px 5px 20px rgb(74, 74, 74);
+  color: white;
+  padding: 10px;
+  margin: 5px;
+  border-radius: 0px 25px;
+}
```

You have two options here. Either have the button styles in the corresponding components or extract a base style for your buttons and create modifier with the rest of the styles. 
```suggestion
.button {
  background-color: rgb(134, 134, 134);
  box-shadow: 0px 5px 20px rgb(74, 74, 74);
  color: white;
  padding: 15px;
  margin: 5px;
  border-radius: 15px;
}

.button--address {
  background-color: rgb(86, 107, 88);
}

.button--footer {
  background-color: rgb(134, 134, 134);
  padding: 10px;
  margin: 5px;
  border-radius: 0px 25px;
}
```
Just so you don't need to repeat the styles all the time.